fa25-team-b/cleaners/race_naming_normalization.py

```
@@ -0,0 +1,381 @@
+"""
+Created on Mon Oct  6 19:35:04 2025
+
+@author: brend
+
+EDITS:
+    kristenbestavros - wrapped Brendan's work in a function so it can be called externally.
+"""
+import pandas as pd
+import re
+
+def normalize_race(df: pd.DataFrame, col: str = "Race/Ethnicity") -> pd.DataFrame:
```

Ruff (0.14.3)

Unused function argument: `col`

(ARG001)